### ISSUE TITLE
TypeScript definitions: "exclude" should be "excludeLicenses"

### ISSUE THREAD

**Issue by personalizedrefrigerator** [RSeidelsohn/license-checker-rseidelsohn#101]
# Issue

`excludeLicenses` isn't listed as an option in `index.d.ts` and is instead listed as `exclude`:

https://github.com/RSeidelsohn/license-checker-rseidelsohn/blob/20d383f79698ac14e3630a4a9d47655e7049c83d/lib/index.d.ts#L48

`index.js` seems to only accept `excludeLicenses`, and require that `excludeLicenses` be a `string|undefined`, rather than a `string[]`:

https://github.com/RSeidelsohn/license-checker-rseidelsohn/blob/20d383f79698ac14e3630a4a9d47655e7049c83d/lib/index.js#L440-L442

Similarly, if `excludeLicenses` is given as an array of `string`s, `license-checker-rseidelsohn` fails with the error,
```
/home/username/path/to/project/node_modules/license-checker-rseidelsohn/lib/index.js:442
                .match(/([^\\\][^,]|\\,)+/g)
                 ^
```

# Suggested fix

I suggest replacing
```
    exclude?: string[] | undefined; 
```
with
```
    excludeLicenses?: string | undefined; 
```

Thank you for maintaining this very useful project!

**Comment by RSeidelsohn** [comment]
Than you for your comment and suggestion. Will probably be up this weekend.

**Comment by RSeidelsohn** [comment]
Hi @personalizedrefrigerator,

I now released a maintenance version (v4.2.11) that includes your fix.

Cheers, Roman.

**Comment by personalizedrefrigerator** [comment]
Thank you!
